lib/game.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+  attr_reader :board, :player_o_turn
+
+  def initialize
+    @board = Board.new
+    @player_x = Player.new
+    @player_o = Player.new(name = 'Player O', tag = 'O')
+    @player_o_turn = false
+  end
+
+  def play
+    switch_player
+    player_turn
+    request_move
+    x, y = get_move
+    @player_o_turn ? tag = 'O' : tag = 'X'
```

Comment:
DRY: It feels like knowledge of the tag for each player is duplicated in `Player` and here.